### DISCUSSION TITLE
2005/09/28/msn-space-%E6%8A%80%E6%9C%AF%E6%96%87%E7%AB%A0%E6%89%80%E5%BC%95

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#50] [Announcements]
# 2005/09/28/msn-space-%E6%8A%80%E6%9C%AF%E6%96%87%E7%AB%A0%E6%89%80%E5%BC%95

本页一共被访问了：

https://blog.qizhen.xyz/2005/09/28/msn-space-%E6%8A%80%E6%9C%AF%E6%96%87%E7%AB%A0%E6%89%80%E5%BC%95

**Comment by ruanqizhen** [comment]
Qizhen
2006年09月9日 19:48 编辑
从这篇总结贴写完后，到现在，msn space 改名叫了 Windows Live Space，功能也增加了不少。但是现在不像当时了，网上总结 Live Space 技术的网站已经非常多了，所以我觉得我也没必要再花精力来更新这里的内容了。对于 Live Space 基本用法，这里看看还是适用的。对于新的高级用法，这里并不全面，要去别的网站查找。

回复
lei
2006年09月9日 17:37 编辑
老大贴的东西都很有用呢！

回复
飞翔的雄鹰
2006年08月10日 21:40 编辑
新手正在寻找space技术资料呢，这么全，谢谢了！

回复
